# HG changeset patch
# User ttl <ttl@justmail.de>
# Date 1312040205 -7200
#      Sat Jul 30 17:36:45 2011 +0200
# Node ID 316108b2dcd78db72940f675b8c8258d179be2e2
# Parent  248b897d9f3639b6ef33720da6ce2e917d9e71ff
edtior: better comments in lexer files

diff --git a/gui/src/lexer/lexeroctavegui.cpp b/gui/src/lexer/lexeroctavegui.cpp
--- a/gui/src/lexer/lexeroctavegui.cpp
+++ b/gui/src/lexer/lexeroctavegui.cpp
@@ -1,16 +1,27 @@
-// This module is a custom lexer for QScintilla for use with the Octave-GUI.
-// Examples for lexers can be found in the sources of Qscintilla-gpl-2.5.1,
-// QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
+/* OctaveGUI - A graphical user interface for Octave
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * This module is a custom lexer for QScintilla for use with the Octave-GUI.
+ * Examples for lexers can be found in the sources of Qscintilla-gpl-2.5.1,
+ * QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control.
+ */
 
 
 #include "lexeroctavegui.h"
 #include <qcolor.h>
 #include <qfont.h>
 
 // -----------------------------------------------------
 // Some basic functions
diff --git a/gui/src/lexer/lexeroctavegui.h b/gui/src/lexer/lexeroctavegui.h
--- a/gui/src/lexer/lexeroctavegui.h
+++ b/gui/src/lexer/lexeroctavegui.h
@@ -1,16 +1,27 @@
-// This module is a custom lexer for QScintilla for use with the Octave-GUI.
-// Examples for lexers can be found in the sources of Qscintilla-gpl-2.5.1,
-// QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU General Public License for more details.
+/* OctaveGUI - A graphical user interface for Octave
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * This module is a custom lexer for QScintilla for use with the Octave-GUI.
+ * Examples for lexers can be found in the sources of Qscintilla-gpl-2.5.1,
+ * QScintilla is a port to Qt of Neil Hodgson's Scintilla C++ editor control.
+ */
 
 
 #ifndef LEXEROCTAVE_H
 #define LEXEROCTAVE_H
 
 #include <qobject.h>
 
 #include <Qsci/qsciglobal.h>
